includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -142,6 +142,16 @@ public function register_endpoints() {
 				'permission_callback' => [ $this, 'generate_post_title_permissions_check' ],
 			]
 		);
+
+		register_rest_route(
+			'classifai/v1/openai',
+			'resize-content',
+			[
+				'methods'             => WP_REST_Server::EDITABLE,
```

Comment:
Does this need to be `EDITABLE` or can we use `CREATABLE`, which will only support `POST` requests?